webapp/lib/view.dart

```diff
@@ -211,26 +212,40 @@ class ConfigurationViewTagListPartial {
       ..classes.add('configure-package__tags-list');
   }
 
-  void renderTagList(List<String> tags) {
+  void renderTagList(Map<String, bool> tags) {
     tagListElement.children.clear();
-    tags.forEach((tag) {
+    tags.forEach((tag, state) {
       var tagItem = new Element.li()
         ..classes.add('configure-package__tag-item')
         ..text = tag;
       tagItem.onClick.listen((event) {
         var selectedTag = (event.target as Element);
         controller.command(controller.UIAction.configurationTagSelected, new controller.ConfigurationTagData(selectedTag: selectedTag.text.trim()));
-        tagListElement.children.forEach((t) => t.classes.remove('configure-package__tag-item-active'));
-        selectedTag.classes.add('configure-package__tag-item-active');
+      });
+      tagItem.onDragEnter.listen((event) => event.preventDefault());
+      tagItem.onDragOver.listen((event) {
+        event.preventDefault();
+        (event.target as Element).style.backgroundColor = '#b5b3b3';
+      });
+      tagItem.onDragLeave.listen((event) => (event.target as Element).style.backgroundColor = '#ffffff');
+      tagItem.onDrop.listen((event) {
+        event.preventDefault();
+        var dropTarget = (event.target as Element);
+        dropTarget.style.backgroundColor = '#ffffff';
```

Comment:
we shouldn't try to directly set css values here - we should use css classes instead

Reply:
this applies to all the other places in this file, esp. around `backgroundColor`

Reply:
Adding the CSS classes doesn't show the desired CSS effect instantly. I don't know why. That's why I'm applying the CSS directly.